data/default_ocp_requirements.json

```diff
@@ -0,0 +1,15 @@
+[{
+  "version": "default",
```

Comment:
What does it says default?
Could we have "SNO" version here as well? I thought it's only OCP semver notation.

Reply:
It's a fallback requirements definition and has special support in the code - maintaining duplicate requirements for each and every version would be terrible:)